benchmarks/build-system-cost/main.go

```
@@ -0,0 +1,679 @@
+		}
+	}
+	goSummary, err := summarizeArm(byArm[armGo])
+	if err != nil {
+		return summary{}, err
+	}
+	bazelSummary, err := summarizeArm(byArm[armBazel])
+	if err != nil {
+		return summary{}, err
+	}
+	if bazelSummary.CleanBuildSeconds == 0 || bazelSummary.WarmBuildSeconds == 0 || bazelSummary.EditBuildSeconds == 0 {
+		return summary{}, errors.New("Bazel median is zero; cannot calculate ratios")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)